zammad/values.yaml

```diff
@@ -55,6 +55,15 @@ envConfig:
     nginx:
       livenessProbe: true
       readinessProbe: true
+
+      # string with nginx config to set headers on the /ws location
+      # websocketExtraHeaders: |
+      #   proxy_set_header Header1 "Header Value";
+
+      # string with nginx config to set headers on the / location
+      # extraHeaders: |
+      #   proxy_set_header Header2 "Header2 Value";
+
```

Comment:
Commented out stuff looks bad.
Please only comment out the value example but add an empty key.

Reply:
I modeled this on: https://github.com/zammad/zammad-helm/blob/1fe1b50f7a687ba3178bc83bc5a944747059a8c4/zammad/values.yaml#L69

Are you suggesting I remove line 63, add an "extraHeaders: {}" and keep 64+65 as an example?

Reply:
yes